FND/GoalsAndObjectives/Objectives.rdf

```diff
@@ -109,6 +110,17 @@
 		<skos:definition>short-term, quantitative, measurable result that a person, organization, or system seeks to attain in order to achieve its long-term goals</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fnd-gao-obj;Program">
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-gao-obj;hasObjective"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-gao-obj;Objective"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>program</rdfs:label>
+		<skos:definition>coordinated set of activities designed to achieve some objective(s)</skos:definition>
+	</owl:Class>
+	
 	<owl:Class rdf:about="&fibo-fnd-gao-obj;SalesStrategy">
```

Comment:
(aside) I don't know why on earth we have this. In any case things like resource allocation are part of tactics not strategy

Reply:
This isn't about general resource allocation, only about allocation of resources specific to the sales process.  I've eliminated the 'or' however.